tests/test_core.py

```diff
@@ -124,14 +124,14 @@ def test_findlib2():
         os.mkdir(fi_dir)
     dirs, paths = core.findlib.generate_candidate_libs(["libfreeimage"], [fi_dir])
     # assert fi_dir in dirs -> Cannot test: lib may not exist
-    assert paths
+    # assert paths
 
     open(os.path.join(fi_dir, "notalib.test.so"), "wb")
 
     # Loading libs
     gllib = ctypes.util.find_library("GL")
     core.load_lib([gllib], [])
-    core.load_lib([], ["libfreeimage"], [fi_dir])
+    # core.load_lib([], ["libfreeimage"], [fi_dir])
```

Comment:
Would be good to eventually refactor this test a bit. But we can leave that for another day, e.g. when we roll the freeimage out into its own package, like we did with imageio-ffmpeg (which is something worth doing, I think, because it is the only thing being not pure Python in imageio).

For now, I'll just delete the comment.